server/monitor/src/main/scripts/format-js.sh

```diff
@@ -0,0 +1,27 @@
+#! /usr/bin/env bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# This script will attempt to format JavaScript files
+
+if hash npx 2>/dev/null; then
+  npx js-beautify -r -n -j -s 2 src/main/resources/org/apache/accumulo/monitor/resources/js/*.js
```

Comment:
I'd appreciate any feedback on whether we should be using any additional formatting options on the command line here.

Reply:
I can test this out locally.

Will this change be able to run on any Github Action or Jenkins jobs?

Reply:
We could potentially add `-w, --wrap-line-length            Wrap lines that exceed N characters [0]`. I think things look fine as is but I wanted to point it out since we do have a line wrap set for the java code.

Reply:
> Will this change be able to run on any Github Action or Jenkins jobs?

I believe npx should be installed on the Ubuntu images during GitHub Actions, but it won't run because I defined it in a profile, and we have `-DskipFormat` set on GitHub Actions tasks. I did not set up a corresponding verifyFormat profile, since that would be a bit more complicated, and I didn't want to assume developers had `npx` available (or had a particular version of js-beautify installed).

It might run in Jenkins, I'm not sure. But nothing will be checked in.

Reply:
OK. FYI I had to install `npm` and then setup `npx` module to get it to work on my Ubuntu 18

Reply:
> We could potentially add `-w, --wrap-line-length Wrap lines that exceed N characters [0]`. I think things look fine as is but I wanted to point it out since we do have a line wrap set for the java code.

I looked into this, but it wrapped some things very oddly. I think it'd be better to just wrap manually, if necessary.